code/modules/mob/living/living.dm

```diff
@@ -749,19 +749,25 @@
 // Override if a certain mob should be behave differently when placing items (can't, for example)
 /mob/living/stripPanelEquip(obj/item/what, mob/who, where, var/silent = 0)
 	what = get_active_hand()
-	if(what && (what.flags & NODROP))
-		to_chat(src, "<span class='warning'>You can't put \the [what.name] on [who], it's stuck to your hand!</span>")
-		return
 	if(what)
+		if(what.flags & NODROP)
```

Comment:
why...did you change this.

Reply:
I don't like double checks if something is there. So I moved it in the check where we know it's not nulll

Reply:
Unless there's an actual code reason for this, please revert it.